og_ui/og_ui.services.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  access_check.og.user_access_group:
```

Comment:
I wonder whether this should be rather part of OG itself, so other people can build up their own UI around OG